packages/tabs/src/TabTitle/TabTitle.types.ts

```diff
@@ -1,4 +1,7 @@
+import { PolymorphicAs } from '@leafygreen-ui/polymorphic';
+
 export interface BaseTabTitleProps {
+  as?: PolymorphicAs;
```

Comment:
Do we need to explicitly declare this prop? I believe it gets automatically added by `InferredPolymorphic`

Reply:
oops yeah this is a remnant from earlier